looplab/eventhorizon
# Replace Wercker with Travis and improve Dev/CI

# Summary

Replace Wercker with Travis and improve CI.

This PR replaces Wercker with Travis and streamlines how local and CI builds are done. Furthermore, it should be a bit easier to set up a new eh dev environment with one command :-)

See: https://travis-ci.org/fixl/eventhorizon/builds

# How to test

```
make dep test
make dev
```

after running `make dev`, see that your local development still works

# Issue

Fixes #200 .

# Notes

Things @maxekman  has to do:

  1. Link `github.com/looplab/eventhorizon` to travis-ci.com
  2. Add coveralls repo token to travis as **secret** environment variables:
    - Name: `COVERALLS_TOKEN` (Same one as currently configured on Wercker)
